packages/app-builder/src/components/ui/header/PageSelector.tsx

```diff
@@ -0,0 +1,39 @@
+import { cx } from '@linaria/core'
+import { Button, elFlex, elFlex1, elFlexAlignCenter, elFlexJustifyStart, elM1, Select } from '@reapit/elements'
+import React from 'react'
+import { getPages, newPage } from './saveState'
+
+export const PageSelector = ({ pageId, onChange }: { pageId?: string; onChange: (id: string) => void }) => (
+  <div className={cx(elFlex1, elFlex, elFlexAlignCenter, elFlexJustifyStart)}>
+    <Select
+      style={{ maxWidth: 150 }}
+      id="page_selector"
+      name="page_selector"
+      value={pageId}
+      onChange={(e) => {
+        onChange(e.target.value)
+      }}
+    >
+      {getPages()
+        .map((p) => ({ value: p.id, label: p.name }))
+        .map(({ value, label }) => (
```

Comment:
double map? The below should rename the params 

```suggestion
        .map(({ id: value, name: label }) => (
```